### ISSUE TITLE
An usable Changelog is missing

### ISSUE THREAD

**Issue by renchap** [yarnpkg/yarn#6448]
**Do you want to request a *feature* or report a *bug*?**
Feature

**What is the current behavior?**
Yarn does not have a reliable / proper / up-to-date changelog

**If the current behavior is a bug, please provide the steps to reproduce.**
I can not know what are the major / breaking changes in the latest yarn versions

**What is the expected behavior?**
Using either a CHANGELOG.md file or the Github release page (ideally both), I would like to be able to know what are the major changes in new versions, as well as the (potentially) breaking ones.
This is missing for the last published versions, and for some others (like 1.10.0) a mere list of commits, which is not very useful as an overview.
This should be part of the release process. For example, Babel has a nice (and semi-automated) way of doing this: https://github.com/babel/babel/releases/tag/v7.1.0

**Comment by Daniel15** [comment]
Some changelogs are here: https://github.com/yarnpkg/yarn/releases. However, it looks like not every release has a changelog.

**Comment by renchap** [comment]
Correct, and the changelogs are not very useful when present (see [v1.10.0](https://github.com/yarnpkg/yarn/releases/tag/v1.10.0), as described above.

It should be part of the release process.

**Comment by olingern** [comment]
Hmm, should every merged PR make a commit to update a `CHANGELOG` pre-merge? 

Seems like it would be more difficult for maintainers to do it themselves.

**Comment by arcanis** [comment]
Yeah I think we'll start doing a `CHANGELOG.md`.

**Comment by renchap** [comment]
https://keepachangelog.com/en/1.0.0/ has good guidelines for a CHANGELOG file.
I would also advise to edit it slightly to add a summary for the version before releasing, this really helps users to understand what happened during the release without reading the whole entry (and often needing to look at the linked PRs)

**Comment by arcanis** [comment]
#6460

**Comment by the-spyke** [comment]
Please do not make GitHub releases without release notes. Like seriously.

**Comment by the-spyke** [comment]
@arcanis Which is missing released today `1.14.0`
